rtengine/camconst.json

```diff
@@ -2185,6 +2185,16 @@ Camera constants:
         ]
     },
 
+    { // Quality X
+        "make_model": ["OM Digital Solutions OM-5"],
+        "dcraw_matrix": [ 11896, -5110, -1076, -3181, 11378, 2048, -519, 1224, 5166 ], // ColorMatrix2 using illuminant D65 from Adobe DNG Converter 15.0
+        "raw_crop" : [
+           { "frame" : [8200, 6132], "crop": [0, 0, 8172, 6132] },
+           { "frame" : [5240, 3912], "crop": [0, 0, 5240, 3912] },
+           { "frame" : [10400, 7792], "crop": [0, 0, 10392, 7792] }
```

Comment:
There is still 2 pixels of garbage values on the right border of the 80 MP image.
![image](https://user-images.githubusercontent.com/45837045/227744073-57c58ea7-4da7-4b14-ba12-367520d4cdcb.png)